.gitignore

```diff
@@ -1,15 +1 @@
-# dependencies
-/node_modules
-
-# testing
-/coverage
-
-# production
-/build
-
-# misc
-.DS_Store
```

Comment:
We could probably keep this one. Also, I like having the little section comments to keep the .gitignore organized, since we'll probably add things in the future, so I'd keep the ones for `# dependencies` and `# misc`

Reply:
Yes, indeed. This was just in the wrong place. We need the .gitignore in the /playground folder